element-connector/src/main/java/org/eclipse/californium/elements/EndpointContextUtil.java

```diff
@@ -61,4 +61,33 @@ public static boolean match(String name, Set<String> keys, EndpointContext conte
 		}
 		return matchAll;
 	}
+
+	/**
+	 * Get follow-up endpoint context.
+	 * 
+	 * Using {@link DtlsEndpointContext#KEY_HANDSHAKE_MODE} requires to adjust
+	 * the reuse of the endpoint context in case of blockwise transfers or
+	 * retransmissions.
+	 * 
+	 * @param messageContext messages's endpoint context.
+	 * @param connectionContext the connection's endpoint context. Either
+	 *            reported with
+	 *            {@link RawData#onContextEstablished(EndpointContext)} or
+	 *            contained in a received message.
+	 * @return endpoint context to be used for follow-up messages.
+	 * @since 2.3
+	 */
+	public static EndpointContext getFollowUpEndpointContext(EndpointContext messageContext,
```

Comment:
clearly a detail but I would change the name to make clear this is for block. (even if this is already mention in the javadoc)

Reply:
It's also for "retransmissions of CONs". 
So I'm not sure, which name you have in mind?

Reply:
correct forget it !